doc/maintaining/installing/install-from-package.rst

```diff
@@ -57,7 +57,7 @@ CKAN:
 
 #. Download the CKAN package:
 
-    - On Ubuntu 16.04:
+    - On Ubuntu 18.04:
```

Comment:
Rather than replace the version we need two entries, something like:

```
   - On Ubuntu 16.04:

       .. parsed-literal::

           wget \http://packaging.ckan.org/|latest_package_name_xenial|

   - On Ubuntu 18.04:

       .. parsed-literal::

           wget \http://packaging.ckan.org/|latest_package_name_bionic|


```

The `latest_package_name_bionic` needs defining [here](https://github.com/ckan/ckan/blob/master/doc/conf.py#L310), using `bionic` as distro name.